src/pivotal-ui/components/forms/forms.scss

```diff
@@ -1439,11 +1443,29 @@ For an organization name input:
     <p class="mtl">https://www.npmjs.com/org/<strong>your-org-name</strong></p>
   </div>
 </div>
+<div class="form-group">
+  <label for="example">Example for error handling</label>
+  <input name="example" id="example-for-error-handling-2" class="form-control">
+</div>
 ```
 */
 
 .link-updater-container {
+  transition: margin-bottom .1s;
+
+  .link-updater {
+    position: relative;
+  }
   p {
     color: $form-label-color;
   }
+
+  &.has-error {
+    margin-bottom: 35px;
+  }
+
+  .has-error {
```

Comment:
One is `has-error` on the container itself, one is `has-error` on the error block. This seems to fit the convention for error handling I've seen elsewhere in PUI.